content/en/guides/deployment/deployment-azure-portal.md

```diff
@@ -0,0 +1,171 @@
+---
+title: How to deploy on Azure Portal?
+description: How to deploy a Nuxt.js application on Azure Portal?
+menu: Deploy on Azure Portal
+category: deployment
+position: 202
+---
+
+## Requirements
+
+- It is required that you select a backend when setting up the project. Even if you don't need it, or else the site won't start up.
+- The server is running Node 8 or greater
+
+## What if I already have a project without an backend?
+
+No worries. It is easy to add an express server to an existing project.
+
+Create a new folder called `server` in the root of the project. Then create an `index.js` file inside the `server` folder and paste the following inside the `index.js`:
+
+```
+const express = require('express')
+const consola = require('consola')
+const { Nuxt, Builder } = require('nuxt')
```

Comment:
This usage is wrong (we should use `loadNuxt`/`build` API from nuxt package), then express and manually loading config is not needed

Reply:
can you give example as not sure I fully understand. thanks

Reply:
https://nuxtjs.org/docs/2.x/internals-glossary/nuxt/ and basically we don't need builder / full nuxt (since entrypoint is meant to use for production only):

```js
const { loadNuxt, build } = require('nuxt-start')

async function start () {
  const nuxt = await loadNuxt(isDev ? 'dev' : 'start')
  await nuxt.listen(process.env.PORT, process.env.HOST)
}

start()
```

/cc @danielroe If I'm missing anything for azure-portal target

Reply:
updated. let me know if it is ok @pi0 or if I deleted too much